pages/api/v1/posts/comments/subsquare-comments.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import { v4 as uuid } from 'uuid';
+
+const urlMapper:any= {
+	bounties: (id: any, network: string) => `https://${network}.subsquare.io/api/treasury/bounties/${id}/comments`,
+	child_bounties: (id: any, network: string) => `https://${network}.subsquare.io/api/treasury/child-bounties/${id}/comments`,
+	council_motions: (id: any, network: string) => `https://${network}.subsquare.io/api/motions/${id}/comments`,
+	democracy_proposals: (id: any, network: string) => `https://${network}.subsquare.io/api/democracy/proposals/${id}/comments`,
+	fellowship_referendums: (id: any, network: string) => `https://${network}.subsquare.io/api/fellowship/referenda/${id}/comments`,
+	referendums: (id: any, network: string) => `https://${network}.subsquare.io/api/democracy/referendums/${id}/comments`,
+	referendums_v2: (id: any, network: string) => `https://${network}.subsquare.io/api/gov2/referendums/${id}/comments`,
+	tech_committee_proposals: (id: any, network: string) => `https://${network}.subsquare.io/api/tech-comm/motions/${id}/comments`,
+	tips: (id: any, network: string) => `https://${network}.subsquare.io/api/treasury/tips/${id}/comments`,
+	treasury_proposals: (id: any, network: string) => `https://${network}.subsquare.io/api/treasury/proposals/${id}/comments`
+};
+
+const characters ='ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+
+const generateRandomString = (length:number) => {
+	let result = '';
+	const charactersLength = characters.length;
+	for ( let i = 0; i < length; i++ ) {
+		result += characters.charAt(Math.floor(Math.random() * charactersLength));
+	}
+
+	return result;
+};
+
+const getTrimmedUsername = (username:string) => {
+	if(!username){
+		return generateRandomString(8);
+	}
+	return username.length >= 8 ? username?.slice(0, 8) :username;
+};
+
+const getReactionUsers = (reactions:any) => {
+	return reactions.map((rec:any) => {
+		return rec.user?.address || '';
```

Comment:
![image](https://user-images.githubusercontent.com/65517966/234763421-4bd559ad-bea6-4a98-a3a0-2965f5ce1ab7.png)

if I'm removing empty strings it gives me an error because the user is now undefined...